Level 1. Lesson 2. Написал все программы на Go.

t88code opened the pull request:
Lesson 2 is ready
https://gb.ru/lessons/182478/homework

t88code committed:
Create main.go
Lesson 2 is ready

goarchitecture commented on lesson-2/main.go:
```diff
@@ -0,0 +1,105 @@
+	}
+}
+
+func calcAreaRectangle() {
```
пусть ваша функция возвращает ошибку. Тогда не нужно быть выходить из прогарммы при ошибках, а просто возвращать её

goarchitecture commented on lesson-2/main.go:
```diff
@@ -0,0 +1,105 @@
+	fmt.Println("Выбрана программа для вычисления диаметра и длины окружности по заданной площади круга")
+
+	var d, l, s float64
+	pi := math.Pi
```
не зачем вывводить в отдельную переменную

goarchitecture reviewed (commented):
отлично. доработакйте по желанию по комментариям

t88code committed:
Правки:
1. Все функции возвращают ошибку в main.
2. os.Exit(1) нигде не используется.
3. Отдельно выделенная переменная pi удалена. Вместо нее в вычислениях используется значение переменной ПИ, полученное напрямую из пакета math - math.Pi.

t88code committed:
Fix line 19
